src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -13161,5 +13161,564 @@ .maxstack  1
     IL_0002:  ret
 }");
         }
+
+        [Theory]
+        [InlineData("void M1() { }", 0)]
```

Comment:
>void [](http://example.com/codeflow?start=21&length=4)

Consider testing deleting constructor, property and event accessors, an operator. These cases should be the same as other methods to the compiler.